app/javascript/packs/component-definitions-common.js

```diff
@@ -155,7 +155,11 @@ import HostInitiatorGroupForm from '../components/host-initiator-group-form';
 import StorageServiceForm from '../components/storage-service-form';
 import VolumeMappingForm from '../components/volume-mapping-form';
 import ZoneForm from '../components/zone-form';
+<<<<<<< HEAD
```

Comment:
@anu-jose-ibm , please fix this so that we can merge this.

Reply:
Done